locales/en-US/pocket.properties

```diff
@@ -26,6 +26,8 @@ tagline = Save articles and videos from Firefox to view in Pocket on any device,
 taglinestory_one = Click the Pocket Button to save any article, video or page from Firefox.
 taglinestory_two = View in Pocket on any device, any time.
 tagssaved = Tags Added
+tos = By continuing, you agree to Pocket's <a href="https://getpocket.com/tos?s=ffi&t=tos&tv=panel_tryit">Terms of Service</a> and <a href="https://getpocket.com/privacy?s=ffi&t=privacypolicy&tv=panel_tryit">Privacy Policy</a>
```

Comment:
Not in add-ons, but it could be as simple as

`
tos = By continuing, you agree to Pocket's <a href="#1">Terms of Service</a> and <a href="#2">Privacy Policy</a> 
`

And then str.replace the placeholders with the links. Not sure if add-ons have better way to manage variables when accessing strings.